res/controllers/Roland_DJ-505-scripts.js

```diff
@@ -970,17 +970,24 @@ DJ505.PadColor = {
     DIM_MODIFIER: 0x10,
 };
 
-DJ505.PadColorMap = [
-    DJ505.PadColor.OFF,
-    DJ505.PadColor.RED,
-    DJ505.PadColor.GREEN,
-    DJ505.PadColor.BLUE,
-    DJ505.PadColor.YELLOW,
-    DJ505.PadColor.CELESTE,
-    DJ505.PadColor.PURPLE,
-    DJ505.PadColor.APRICOT,
-    DJ505.PadColor.WHITE,
-];
+DJ505.PadColorMap = new ColorMapper({
```

Comment:
Why do we have so many colors here?

Reply:
These are all colors supported by the DJ-505. For any colored hotcue the ColorMapper class will find the closest matching color supported by the controller and use that.

Reply:
Note to myself: Remove the unnecessary alpha channel here